v2/cli/init_command_test.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNormalizeLanguage(t *testing.T) {
+	testCases := []struct {
+		name     string
+		expected string
+	}{
+		{"typescript", "ts"},
+		{"ts", "ts"},
+		{"javascript", "js"},
+		{"js", "js"},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			result, err := normalizeLanguage(testCase.name)
+			require.NoError(t, err)
+			require.Equal(t, result, testCase.expected)
+		})
+	}
+}
+
+func TestIDMatch(t *testing.T) {
+	testCases := []struct {
+		id string
+	}{
+		{"abc"},
+		{"a11"},
+		{"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.id, func(t *testing.T) {
+			err := idMatch(testCase.id)
+			require.NoError(t, err)
+		})
+	}
+
+	wrongTestCases := []struct {
```

Comment:
I'd rather see one set of test cases than divide it up into "good cases and bad cases."

I can pair with you later to show you the pattern for incorporating expected results that vary by case.